esrally/driver/driver.py

```diff
@@ -582,12 +582,18 @@ def __init__(self):
 
     @actor.no_retry("load generator")
     def receiveMsg_StartLoadGenerator(self, msg, sender):
+        def EsClients(all_hosts, all_client_options):
```

Comment:
rename to `es_clients`? (I immediately thought that this is a class when I read the code below that calls this function)

Reply:
Addressed in 0a17876dd048e0720d5104f6d5897d31de5dc94f